src/utils/helpers.js

```diff
@@ -20,17 +19,17 @@ export const normalizeErrors = error => {
 };
 
 /**
- * @returns {undefined}
+ * @returns {object} current user
  */
 export const checkAuth = async () => {
   try {
     const token = localStorage.getItem('token');
     const res = await axios.get(`${process.env.API_BASE_URL}/user`, {
       headers: { authorization: token },
     });
-    store.dispatch(action.authSuccessAction({ user: res.data.user }));
+    return res.data;
```

Comment:
I think I was not clear enough initially - my point with this function was, why was this manually coupled to the store implementation when it could simply be an action dispatcher?

Reply:
Yeah, I agree that's a better way to go about it 🙂